SU2_CFD/include/variables/CIncEulerVariable.hpp

```diff
@@ -101,7 +99,19 @@ class CIncEulerVariable : public CFlowVariable {
     Primitive(iPoint, indices.Density()) = val_density;
     return val_density <= 0.0;
   }
+  
+  inline void Set_Density_time_n(unsigned long iPoint, su2double val) {
+    Density_time_n[iPoint] = val;
+  }
 
+  inline void Set_Density_unsteady(unsigned long iPoint, su2double val) {
+    Density_unsteady[iPoint] = val;
+  }
+  
+  inline su2double GetDensity_time_n(unsigned long iPoint) const {
+    return Density_time_n[iPoint];
+  }
+  
```

Comment:
```suggestion
  
  inline su2double GetDensity_Time_n(unsigned long iPoint) const {
    return Density_time_n[iPoint];
  }
  
```

documentation must be added